images/logging-eventrouter.yml

```diff
@@ -19,3 +19,5 @@ push:
   repos:
   - openshift/logging-eventrouter
   - openshift/ose-logging-eventrouter
+non_shipping_repos:
+- rhel-8-server-ose-rpms
```

Comment:
This doesn't seem right, this image installs openshift-eventrouter which is built as an RPM from this repo.